docs/jitsi.md

```diff
@@ -0,0 +1,47 @@
+# Jitsi in Riot
+
+Riot uses [Jitsi](https://jitsi.org/) for conference calls, which provides options for
+self-hosting your own server and supports most major platforms.
+
+1:1 calls, or calls between you and one other person, do not use Jitsi. Instead, those
+calls work through a TURN server configured on your respective homeservers.
```

Comment:
This might be a bit misleading: maybe, "work directly between clients or via TURN servers configured on the respective homeservers"?